static/docs/user-guide/external-data/local.md

```diff
@@ -0,0 +1,311 @@
+# External Data on a Local Filesystem
+
+Sometimes we would like to keep the data and cache outside the project
```

Comment:
We should mention that it's not a default and recommended way. The recommended setup is the shared cache in this case.

Reply:
> We should mention that it's not a default and recommended way. The recommended setup is the shared cache in this case.

The word "Sometimes" already implies that this is not usual. By the way, this case is not related to the case of "shared cache".